tests/test_notification.py

```diff
@@ -1,10 +1,11 @@
 import unittest
 import logging
 import sys
-from oar.core.notification import NotificationManager
+from oar.core.notification import *
```

Comment:
Why did you update this import?

Reply:
not a big deal, this is test code, you can submit another PR to fix it

Reply:
Done.